From: Michael Froman <mfroman@mozilla.com>
Date: Tue, 14 Feb 2023 03:27:00 +0000
Subject: Bug 1816173 - pt12 - add shim config for
 third_party/libwebrtc/testing/{gmock|gtest} r=ng

We don't vendor third_party/libwebrtc/third_party/gmock
                third_party/libwebrtc/third_party/gtest, so:
- add BUILD.gn to avoid scattered BUILD.gn changes

Differential Revision: https://phabricator.services.mozilla.com/D169674
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/4ea9d2db79c42a144235e45c47c71adf1dd01fdc
---
 testing/gmock/BUILD.gn | 5 +++++
 testing/gtest/BUILD.gn | 5 +++++
 2 files changed, 10 insertions(+)
 create mode 100644 testing/gmock/BUILD.gn
 create mode 100644 testing/gtest/BUILD.gn

diff --git a/testing/gmock/BUILD.gn b/testing/gmock/BUILD.gn
new file mode 100644
index 0000000000..a2a1efdea9
--- /dev/null
+++ b/testing/gmock/BUILD.gn
@@ -0,0 +1,5 @@
+import("//third_party/libaom/options.gni")
+import("../../webrtc.gni")
+
+rtc_library("gmock") {
+}
diff --git a/testing/gtest/BUILD.gn b/testing/gtest/BUILD.gn
new file mode 100644
index 0000000000..c9c2703c37
--- /dev/null
+++ b/testing/gtest/BUILD.gn
@@ -0,0 +1,5 @@
+import("//third_party/libaom/options.gni")
+import("../../webrtc.gni")
+
+rtc_library("gtest") {
+}
